File: trunk/tarah5/www/js/server/LoginService.js

var Connection = jsloader.resolve("server.impl.Connection");

var LoginService = {
	login: function(username, password) {
		var result = false;
		var conn = new Connection("tarah");
		try {
			rs = conn.execute("select * from user where username = ?", [username]);
			if ( rs.fieldByName("password") == password ) {
				result = true;
			} else {
				;
			}
			rs.close();
		} catch(ex) {
			throw new Error("SQL error: " + ex.message);
		} finally {
			try {
				conn.destroy();
			} catch(ex) {
				throw new Error("SQL error: " + ex.message);
			}
		}
		return result;
	}
};

File: trunk/tarah/src/com/honmansoft/tarah/service/LoginService.java
package com.honmansoft.tarah.service;

import java.sql.Connection;
import java.sql.PreparedStatement;
import java.sql.ResultSet;
import java.sql.SQLException;

import com.honmansoft.data.tarah.TarahDBA;
import com.honmansoft.tarah.facade.Login;

public class LoginService implements Login {

	public boolean login(String username, String password) {
		Connection conn = TarahDBA.getConnection();
		try {
			PreparedStatement stmt = conn
					.prepareStatement("select * from user where username = ?");
			stmt.setString(1, username);
			ResultSet rs = stmt.executeQuery();
			if (rs.next()) {
				return rs.getString("password").equals("goodpassword");
			} else {
				return false;
			}
		} catch (SQLException e) {
			e.printStackTrace();
			throw null;
		}
	}

}
